README.md

```diff
@@ -1,141 +1,17 @@
-# Getting started with Liberty on Bluemix
-By following this guide, you'll set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+
+# Liberty on Bluemix getting started application
+The Bluemix Getting started tutorial for Liberty uses this sample application to provide you with a sample workflow for working with any Liberty app on Bluemix; you set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+
+The Liberty app uses a [Cloudant noSQL DB service](https://console.bluemix.net/catalog/services/cloudant-nosql-db) to add information to a database and then return information from a database to the UI. To learn more about how the app connects to Cloudant, see the [Cloudant library for Liberty]().
```

Comment:
This app uses Liberty's mongodb-2.0 feature and MongoDB Java driver. I think we could link to https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_mongodb_create.html and https://mongodb.github.io/mongo-java-driver/.